presto-main/src/main/java/com/facebook/presto/execution/SqlQueryExecution.java

```diff
@@ -189,6 +189,23 @@ public long getTotalMemoryReservation()
     }
 
     @Override
+    public long getTotalRevocableMemoryReservation()
+    {
+        // acquire reference to outputStage before checking finalQueryInfo, because
```

Comment:
what do revokable memory has with outputStage? Do we care about revokable memory reservation when the query finishes?